Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -228,31 +228,28 @@ You can install Cilium on any Kubernetes cluster. Pick one of the options below:
            cilium install |CHART_VERSION|
 
     .. group-tab:: AKS
+       
+       .. note::
+
+          On AKS, Cilium can be installed either manually by administrators (Bring your own CNI) or 
+          automatically by AKS (Azure CNI Powered by Cilium). Bring your own CNI offers more flexibility 
+          and customization as administrators have full control over the installation, but does not 
+          integrate natively with the Azure network stack and administrators need to handle Cilium upgrades. 
+          Azure CNI Powered by Cilium integrates natively with the Azure network stack and upgrades are 
+          handled by AKS, but does not offer as much flexibility and customization as it is controlled by AKS. 
+          The following instructions assume Bring your own CNI, please refer to the Azure CNI Powered by 
+          Cilium external installer guide :ref:`aks_install` for dedicated instructions.
```

Comment:
Minor grammar tidyup:
```suggestion
          The following instructions assume Bring your own CNI. For Azure CNI Powered by 
          Cilium, see the external installer guide :ref:`aks_install` for dedicated instructions.
```

However, the `:ref:` token may substitute a doc title here so you'll want to test out the docs locally and check how the sentence looks.